tests/client/coll.rs

```diff
@@ -14,7 +12,7 @@ fn find_and_insert() {
 
     // Insert document
     let doc = doc! {
-        "title" => Bson::String("Jaws".to_owned())
+        "title" => (Bson::String("Jaws".to_owned()))
```

Comment:
So let's merge it this way for now, and see if there's a way we can improve it later. This way is miles ahead of our current mechanisms for building bson docs :)